src/main/java/org/folio/service/orders/BaseValidationService.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.service.orders;
+
+import org.folio.rest.core.exceptions.ErrorCodes;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public abstract class BaseValidationService {
+  /**
+   * Checks if claiming interval is greater than 0 if claiming is active.
+   *
+   * @param claimingActive is claiming active
+   * @param claimingInterval claiming interval
+   * @return list of error codes
+   */
+  public List<ErrorCodes> checkClaimingConfig(Boolean claimingActive, Integer claimingInterval) {
```

Comment:
Maybe better to use protected access modifier here. Because you are calling this method from subclasses

Reply:
Agree, fixed